cora/util/cubicspline.pyx

```diff
@@ -341,4 +340,4 @@ cdef class SinhInterpolater(Interpolater):
         for i in prange(nr, nogil=True):
             rr[i] = self.f_t * sinh(self.value_cdef(asinh(xr[i] / self.x_t)))
 
-        return rr.reshape(x.shape)
+        return np.asarray(rr).reshape(x.shape)
```

Comment:
You might want to apply the same changes you did to `LogInterpolater.value_log_array` in here.